squash/dashboard/views.py

```diff
@@ -177,10 +180,15 @@ def embed_bokeh(request, bokeh_app):
     bokeh_script = autoload_server(None, app_path="/{}".format(bokeh_app),
                                    url=bokeh_url)
 
+    template = loader.get_template('dashboard/embed_bokeh.html')
+
     context = {'bokeh_script': bokeh_script,
                'bokeh_app': bokeh_app}
 
-    return render(request, "dashboard/embed_bokeh.html", context)
+    response = HttpResponse(template.render(context, request))
+    response.set_cookie('django_full_path', request.get_full_path())
```

Comment:
A new configuration for the production deployment was applied https://github.com/lsst-sqre/sandbox-jenkins-demo/pull/91